api/scripts/certification/next-gen/import-pilot-certification-centers-from-csv.js

```diff
@@ -2,18 +2,18 @@ import 'dotenv/config';
 
 import lodash from 'lodash';
 
-import { disconnect, knex } from '../../db/knex-database-connection.js';
-import { logger } from '../../src/shared/infrastructure/utils/logger.js';
+import { disconnect, knex } from '../../../db/knex-database-connection.js';
+import { logger } from '../../../src/shared/infrastructure/utils/logger.js';
 /**
  * Usage: node scripts/certification/import-pilot-certification-centers-from-csv.js path/file.csv
```

Comment:
modifier le chemin du script

```suggestion
* Usage: node scripts/certification/next-gen/import-pilot-certification-centers-from-csv.js path/file.csv
```

Reply:
Done !